Src/Couchbase.Linq.IntegrationTests/Couchbase.Linq.IntegrationTests.csproj

```diff
@@ -24,6 +24,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <Prefer32Bit>false</Prefer32Bit>
+    <LangVersion>5</LangVersion>
   </PropertyGroup>
```

Comment:
Ah, LangVersion attribute...this is nice, but probably should be a separate commit.